### DISCUSSION TITLE
How are "Net Worth" and "Net Investment" calculated?

### DISCUSSION THREAD

**Discussion by the-solipsist** [ananthakumaran/paisa#172] [Q&A]
I thought "Net Worth" would be `Assets:* - Liabilities:*` (or what is provided by the `hledger bs -X₹` report).

For investments, since I have all amounts I have invested under `Assets:Investments:*`, I thought that would be what would be counted as invested amounts.

But those are far from the figures I see on the dashboard. A search through the documentation didn't shed any light on this.

**Comment by ananthakumaran** [comment]
> I thought "Net Worth" would be Assets:* - Liabilities:* (or what is provided by the hledger bs -X₹ report).

This is correct, let me know if this is not the case. A minimal ledger example would be helpful

Net investment has some issues and might not be correct if you have checking accounts, I am exploring alternate approaches.

**Reply by the-solipsist** [reply]
I mean to get around to figuring out where it's going wrong, with a minimal hledger example.  

But I think I managed to figure out at least one difference between the output of `hledger` and `paisa`.

With `hledger`, you have have account types.  So, that way, you can have an "asset" account that doesn't begin with "Assets:\*".  (For some retirement assets, which I cannot access until I'm 60, I use "Retirement:Assets:\*".  I think `paisa` disregards those.

Separately, is there a way that I can get `paisa` to use the prices database I use with hledger? I appended all the prices to the paisa.journal, but that didn't have any impact.

**Reply by ananthakumaran** [reply]
Paisa is strict with naming conventions and will anything outside `Assets:*`

try to run pricedb command on the journal, if it returns price history, it will be used by paisa as well.
https://github.com/ananthakumaran/paisa/blob/de8874184db48535c8b24dbc82c309c36b0fdc67/internal/ledger/ledger.go#L118
